### PR TITLE
Fix configmanagement opname exceptions and inconsistencies for integration tests

### PR DESCRIPTION
**Pull request by pedrottimark**
## Description

TL;DR pattern for opname of GraphQL queries:

* plural: roles for /main/configmanagement/roles
* singular: getRole for:
    * side panel: /main/configmanagement/roles/id
    * single page: /main/configmanagement/role/id
* combined: getRoleSUBJECT for:
    * side panel: /main/configmanagement/roles/id/subjects
    * single page: /main/configmanagement/role/id/subjects

### Analysis of current behavior

| path segment    | plural exception | singular entity   | prefix exception   | suffix          |
| :---            | :---             | :---              | :---               | :---            |
| clusters        |                  | getCluster        | getCluster_        | CLUSTER         |
| controls        |                  | controlById       |                    | CONTROL         |
| deployments     | getDeployments   | getDeployment     |                    | DEPLOYMENT      |
| images          |                  | getImage          | getImage_          | IMAGE           |
| namespaces      |                  | getNamespace      | getNamespace_      | NAMESPACE       |
| nodes           |                  | getNode           |                    | NODE            |
| policies        |                  | getPolicy         | getPolicy_         | POLICY          |
| roles           | k8sRoles         | k8sRole           | getRole_           | ROLE            |
| secrets         |                  | getSecret         | getSecret_         | SECRET          |
| serviceaccounts | serviceAccounts  | getServiceAccount | getServiceAccount_ | SERVICE_ACCOUNT |
| subjects        |                  | getSubject        | subject_           | SUBJECT         |

### Primary exceptions in current behavior

1. To intercept and wait for plural requests successfully, I tried most possibilities manually to find exceptions to path segment as opname.

    * The 3 **plural exceptions** require an extra map for the test helper function:

        ```js
        function getRequestConfigForEntities(entitiesKey) {
            const opname = opnameExceptionsForEntities[entitiesKey] ?? entitiesKey;
            return {
                routeMatcherMap: {
                    [opname]: graphql(opname),
                },
            };
        }
        ```

        * `getDeployments` given argument: `deployments` although `getCamelCase` is consistent with Vulnerability Management (too bad, so sad)
        * `k8sRoles` given argument: `roles`
        * `serviceAccounts` given argument: `serviceaccounts`

    * Without exceptions, here is a simpler test helper function:

        ```js
        function getRequestConfigForEntities(entitiesKey) {
            const opname = entitiesKey;
            return {
                routeMatcherMap: {
                    [opname]: graphql(opname),
                },
            };
        }
        ```

        * `deployments` given argument: `deployments`
        * `roles` given argument: `roles`
        * `serviceaccounts` given argument: `serviceaccounts`

2. To intercept and wait for combined requests successfully, I tried most possibilities manually to find exceptions to singular opname as prefix.

    * The 3 **prefix exceptions** require an extra map for the test helper function:

        ```js
        function opnameForPrimaryAndSecondaryEntities(entitiesKey1, entitiesKey2) {
            const opnamePrefix =
                opnamePrefixExceptionForPrimaryAndSecondaryEntities[entitiesKey1] ??
                opnameForEntity[entitiesKey1];
            return `${opnamePrefix}${typeOfEntity[entitiesKey2]}`;
        }
        ```

        * `getCluster_ROLE` given arguments: `clusters` and `roles` with underscore is in the majority
        * `getDeploymentIMAGE` given arguments: `deployments` and `images` without underscore is in the minority
        * `getRole_SUBJECT` given arguments: `roles` and `subjects` has underscore and follows `getCamelCase` pattern, both of which are in the majority, but is inconsistent with singular `k8sRole` opname (see next section).

    * Without exceptions, here is a simpler test helper function:

        ```js
        function opnameForPrimaryAndSecondaryEntities(entitiesKey1, entitiesKey2) {
            return `${opnameForEntity[entitiesKey1]}_${typeOfEntity[entitiesKey2]}`;
        }
        ```

        * `getCluster_ROLE` given arguments: `clusters` and `roles`
        * `getDeployment_IMAGE` given arguments: `deployments` and `images`
        * `getRole_SUBJECT` given arguments: `roles` and `subjects` becomes expected temporary failure

### Secondary inconsistencies in current behavior

Two singular requests have opnames which are inconsistent with getCamelCase pattern of the majority of singular opnames, although consistent with plural opnames.

* controls: `'controlById'` instead of `'getControl'`
* roles: `'k8sRole'` instead of `'getRole'` (see previous section).

### Residue

1. Investigate src/utils/queryMap to evaluate risk of inconsistency or reward for consistency with entity opnames in UI and tests: Compliance, Configuration Management, Vulnerability Management.

## Checklist
- [x] Investigated and inspected CI test results
- [x] Edited integration tests

### manual testing

TODO Compare requests and results on infra demo to staging demo

### CODE REVIEW

**Review by vjwilson — APPROVED** [review]
/lgtm
